### PR TITLE
zimperium - get-report filter HR

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [X] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/26512

## Description
full report still goes to the context. Checked it is ok and does not crush the web client.
hr war room entry is not truncated now, after I filter to the more relevant headders of the report.

## Minimum version of Demisto
- [x] 5.0.0

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

**Comment by bakatzir** [comment]
readme validation is now running:
```
Validating Packs/Zimperium/Integrations/Zimperium/README.md
Packs/Zimperium/Integrations/Zimperium/README.md: [RM100] - Failed verifying README.md Error Message is: SyntaxError: unknown: Expected corresponding JSX closing tag for <br> (9583:160)

  9581 | <tbody parentName="table">
  9582 | <tr parentName="tbody">
> 9583 | <td parentName="tr" {...{"align":null}}>{`sms: `}<br>{`network: {"http_requests": null}`}<br>{`count_sms: 0`}<br>{`telephony: null`}<br>{`broadcast_receivers:`}</td>
       |                                                                                                                                                                 ^
  9584 | <td parentName="tr" {...{"align":null}}>{`f26cf1135f9d2ea60532a5a13c6fbed5`}</td>
  9585 | <td parentName="tr" {...{"align":null}}>{`total: null`}<br>{`status: Waiting in the queue`}<br>{`detected: null`}<br>{`scan_details: `}<br>{`detected_skip: 0`}</td>
  9586 | </tr>
    at Object._raise (/home/circleci/project/node_modules/@babel/core/node_modules/@babel/parser/lib/index.js:742:17)
    at Object.raiseWithData (/home/circleci/project/node_modules/@babel/core/node_modules/@babel/parser/lib/index.js:735:17)
    at Object.raise (/home/circleci/project/node_modules/@babel/core/node_modules/@babel/parser/lib/index.js:729:17)
    at Object.jsxParseElementAt (/home/circleci/project/node_modules/@babel/core/node_modules/@babel/parser/lib/index.js:4588:16)
    at Object.jsxParseElementAt (/home/circleci/project/node_modules/@babel/core/node_modules/@babel/parser/lib/index.js:4556:32)
    at Object.jsxParseElementAt (/home/circleci/project/node_modules/@babel/core/node_modules/@babel/parser/lib/index.js:4556:32)
    at Object.jsxParseElementAt (/home/circleci/project/node_modules/@babel/core/node_modules/@babel/parser/lib/index.js:4556:32)
    at Object.jsxParseElementAt (/home/circleci/project/node_modules/@babel/core/node_modules/@babel/parser/lib/index.js:4556:32)
    at Object.jsxParseElementAt (/home/circleci/project/node_modules/@babel/core/node_modules/@babel/parser/lib/index.js:4556:32)
    at Object.jsxParseElementAt (/home/circleci/project/node_modules/@babel/core/node_modules/@babel/parser/lib/index.js:4556:32) {
  loc: Position { line: 9583, column: 160 },
  pos: 479378,
  code: 'BABEL_PARSE_ERROR'
}
```

File: Tests/scripts/validate.sh [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 set -e
 
+export DEMISTO_README_VALIDATION=yes
```
Comment (yaakovi) [inline Tests/scripts/validate.sh:4]: why do we need this here? `CI` is automatically defined in the build:
image

Comment (bakatzir) [reply Tests/scripts/validate.sh:4]: I thought so too, but the readme was errornous and it passed the build. see in the last pr: https://github.com/demisto/content/pull/7442

**Comment by bakatzir** [comment]
Will fail on the RN vaildation, that's expected.

**Comment by bakatzir** [comment]
Force merging, validation on RN from sdk failed, supposed to be green on master as deleted from the bucket.

**Pull request merged**